src/catalogue/items/catalogueItemsDialog.component.test.tsx

```diff
@@ -148,6 +148,7 @@ describe('Catalogue Items Dialog', () => {
 
   afterEach(() => {
     vi.clearAllMocks();
+    vi.useRealTimers();
```

Comment:
```suggestion
```
You already have this at the end of the test ('displays recently added section')

Reply:
Is it fine to remove it from the end of the test instead? In previous runs when a test failed, it didn't reach the vi.useRealTimers() cmd, and caused all subsequent tests to fail.

Reply:
This will repeatedly runs it after every test when its only needed for the one test - I just tried moving`vi.useRealTimers();` to the end of the `'displays recently added section'` test and the file worked fine for me so I would do that as its just that test that needs to mock/unmock the timers.

Reply:
My main concern is if there's an error before it reaches  `vi.useRealTimers();` For instance if you comment out `vi.advanceTimersByTimeAsync(2000);` this happens.
![image](https://github.com/user-attachments/assets/2bede99c-d66c-41a2-afad-b9782dae94cc)

I tried using a try, finally block in the test to run useRealTimers(), however the test ignores it when it times out. Would it be fine to put this test in its own describe block, and use `afterEach `and `beforeEach` for just this test.

Reply:
Ah yeah good spot, I didn't think of that, yeah I would personally just do that so it has a set of tests with a mocked clock. Same for the e2e tests so that the others can keep the real system time.